src/pdb/mod.rs

```diff
@@ -400,11 +394,11 @@ impl BinRead for RowGroup {
         endian: Endian,
         (page_type, page_heap_offset): Self::Args<'_>,
     ) -> BinResult<Self> {
-        let stream_position = reader.stream_position()?;
+        let table_page_end_position = reader.stream_position()?;
```

Comment:
The new name is more descriptive, but unfortunately wrong if there is more than one row group. It's the end position of the current row group.

Maybe 
```suggestion
        let row_group_end_position = reader.stream_position()?;
```

Reply:
whoops, yeah you're right. Fixup'd